src/java.desktop/share/conf/psfontj2d.properties

```diff
@@ -59,7 +59,6 @@ avantgarde_book_oblique=avantgarde_book_oblique
 avantgarde_demi_oblique=avantgarde_demi_oblique
 #
 itcavantgarde=avantgarde_book
-itcavantgarde=avantgarde_book
```

Comment:
@prrace is it possible that this is a typo and itcavantgarde_book should be used instead?

Reply:
@prrace  Any idea about this code?